src/App.js

```diff
@@ -1,24 +1,38 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import MainPage from './components/pages/Mainpage';
+import HouseDetails from './components/pages/HouseDetails';
+import ReservePage from './components/pages/ReservePage';
+import DeleteHouse from './components/pages/DeleteHouse';
+import Login from './components/auth/Login';
+import Register from './components/auth/Register';
+// import Navigation from './components/sidebar/Navigation';
+import AddHouse from './components/pages/AddHouse';
+import MyReservation from './components/pages/MyReservation';
+// import './styling/style.css';
+// import './styling/navbar.css';
+// import './styling/mainPage.css';
+import './styling/houseReserve.css';
+import setupAxiosInterceptors from './helpers/setupAxiosInterceptors';
+
+setupAxiosInterceptors();
 
 function App() {
```

Comment:
- [ ]  Please, kindly refactor this main `App` component into ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪